common-pg/src/main/scala/com/socrata/pg/soql/SoQLAnalysisSqlizer.scala

```diff
@@ -173,15 +173,44 @@ object SoQLAnalysisSqlizer extends Sqlizer[AnalysisTarget] {
           PostgresUniverseCommon.searchVector(primaryTableReps, Some(ctx))
       }
 
+      val searchNumberLitOpt =
+        try {
+          val number = BigDecimal.apply(searchLit.value)
+          Some(NumberLiteral(number, SoQLNumber.t)(searchLit.position))
```

Comment:
Is `searchLit.position` always going to be `NoPosition` or is there something in the apply of `StringLiteral` on line 165?

Reply:
I don't really care between that and NoPosition.